Unable to build

Perhaps I am doing something wrong.   See 

```
libtool:   error: cannot find the library '../libneurosim/libneurosim.la' or unhandled argument '../libneurosim/libneurosim.la'
Makefile:518: recipe for target 'libpy3neurosim.la' failed
```

during make below

```
$ /usr/bin/libtoolize --install
libtoolize: putting auxiliary files in '.'.
libtoolize: linking file './compile'
libtoolize: linking file './config.guess'
libtoolize: linking file './config.sub'
libtoolize: linking file './depcomp'
libtoolize: linking file './install-sh'
libtoolize: linking file './missing'
libtoolize: linking file './ltmain.sh'
libtoolize: You should add the contents of the following files to 'aclocal.m4':
libtoolize:   '/usr/share/aclocal/libtool.m4'
libtoolize:   '/usr/share/aclocal/ltoptions.m4'
libtoolize:   '/usr/share/aclocal/ltsugar.m4'
libtoolize:   '/usr/share/aclocal/ltversion.m4'
libtoolize:   '/usr/share/aclocal/lt~obsolete.m4'
libtoolize: Consider adding 'AC_CONFIG_MACRO_DIRS([m4])' to configure.ac,
libtoolize: and rerunning libtoolize and aclocal.
libtoolize: Consider adding '-I m4' to ACLOCAL_AMFLAGS in Makefile.am.

$ /usr/bin/autoreconf --force --install
libtoolize: putting auxiliary files in '.'.
libtoolize: copying file './compile'
libtoolize: copying file './config.guess'
libtoolize: copying file './config.sub'
libtoolize: copying file './depcomp'
libtoolize: copying file './install-sh'
libtoolize: copying file './missing'
libtoolize: copying file './ltmain.sh'
libtoolize: putting macros in 'libltdl/m4'.
libtoolize: copying file 'libltdl/m4/libtool.m4'
libtoolize: You should add the contents of 'libltdl/m4/libtool.m4' to 'aclocal.m4'.
libtoolize: copying file 'libltdl/m4/ltargz.m4'
libtoolize: You should add the contents of 'libltdl/m4/ltargz.m4' to 'aclocal.m4'.
libtoolize: copying file 'libltdl/m4/ltdl.m4'
libtoolize: You should add the contents of 'libltdl/m4/ltdl.m4' to 'aclocal.m4'.
libtoolize: copying file 'libltdl/m4/ltoptions.m4'
libtoolize: You should add the contents of 'libltdl/m4/ltoptions.m4' to 'aclocal.m4'.
libtoolize: copying file 'libltdl/m4/ltsugar.m4'
libtoolize: You should add the contents of 'libltdl/m4/ltsugar.m4' to 'aclocal.m4'.
libtoolize: copying file 'libltdl/m4/ltversion.m4'
libtoolize: copying file 'libltdl/m4/lt~obsolete.m4'
libtoolize: You should add the contents of 'libltdl/m4/lt~obsolete.m4' to 'aclocal.m4'.
libtoolize: putting libltdl files in LT_CONFIG_LTDL_DIR, 'libltdl'.
libtoolize: copying file 'libltdl/COPYING.LIB'
libtoolize: creating file 'libltdl/Makefile.am'
libtoolize: copying file 'libltdl/README'
libtoolize: creating file 'libltdl/configure.ac'
libtoolize: creating file 'libltdl/aclocal.m4'
libtoolize: creating file 'libltdl/Makefile.in'
libtoolize: copying file 'libltdl/config-h.in'
libtoolize: creating file 'libltdl/configure'
libtoolize: copying file 'libltdl/libltdl/lt__alloc.h'
libtoolize: copying file 'libltdl/libltdl/lt__argz_.h'
libtoolize: copying file 'libltdl/libltdl/lt__dirent.h'
libtoolize: copying file 'libltdl/libltdl/lt__glibc.h'
libtoolize: copying file 'libltdl/libltdl/lt__private.h'
libtoolize: copying file 'libltdl/libltdl/lt__strl.h'
libtoolize: copying file 'libltdl/libltdl/lt_dlloader.h'
libtoolize: copying file 'libltdl/libltdl/lt_error.h'
libtoolize: copying file 'libltdl/libltdl/lt_system.h'
libtoolize: copying file 'libltdl/libltdl/slist.h'
libtoolize: copying file 'libltdl/loaders/dld_link.c'
libtoolize: copying file 'libltdl/loaders/dlopen.c'
libtoolize: copying file 'libltdl/loaders/dyld.c'
libtoolize: copying file 'libltdl/loaders/load_add_on.c'
libtoolize: copying file 'libltdl/loaders/loadlibrary.c'
libtoolize: copying file 'libltdl/loaders/preopen.c'
libtoolize: copying file 'libltdl/loaders/shl_load.c'
libtoolize: copying file 'libltdl/lt__alloc.c'
libtoolize: copying file 'libltdl/lt__argz.c'
libtoolize: copying file 'libltdl/lt__dirent.c'
libtoolize: copying file 'libltdl/lt__strl.c'
libtoolize: copying file 'libltdl/lt_dlloader.c'
libtoolize: copying file 'libltdl/lt_error.c'
libtoolize: copying file 'libltdl/ltdl.c'
libtoolize: copying file 'libltdl/ltdl.h'
libtoolize: copying file 'libltdl/slist.c'
libtoolize: You should add the contents of the following files to 'aclocal.m4':
libtoolize:   '/usr/share/aclocal/ltargz.m4'
libtoolize: Remember to add 'LTDL_INIT' to configure.ac.
libtoolize: Consider adding 'AC_CONFIG_MACRO_DIRS([libltdl/m4])' to configure.ac,
libtoolize: and rerunning libtoolize and aclocal.
libtoolize: Consider adding '-I m4' to ACLOCAL_AMFLAGS in Makefile.am.
Makefile.am: error: required file './README' not found
Makefile.am: error: required file './ChangeLog' not found
autoreconf: automake failed with exit status: 1

$ touch README ChangeLog

$ /usr/bin/autoreconf --force --install
libtoolize: putting auxiliary files in AC_CONFIG_AUX_DIR, '..'.
libtoolize: copying file '../compile'
libtoolize: copying file '../config.guess'
libtoolize: copying file '../config.sub'
libtoolize: copying file '../depcomp'
libtoolize: copying file '../install-sh'
libtoolize: copying file '../missing'
libtoolize: copying file '../ltmain.sh'
libtoolize: putting macros in 'm4'.
libtoolize: copying file 'm4/libtool.m4'
libtoolize: copying file 'm4/ltargz.m4'
libtoolize: copying file 'm4/ltdl.m4'
libtoolize: copying file 'm4/ltoptions.m4'
libtoolize: copying file 'm4/ltsugar.m4'
libtoolize: copying file 'm4/ltversion.m4'
libtoolize: copying file 'm4/lt~obsolete.m4'
libtoolize: putting libltdl files in LT_CONFIG_LTDL_DIR, '.'.
libtoolize: copying file './COPYING.LIB'
libtoolize: creating file './Makefile.am'
libtoolize: copying file './README'
libtoolize: creating file './configure.ac'
libtoolize: creating file './aclocal.m4'
libtoolize: creating file './Makefile.in'
libtoolize: copying file './config-h.in'
libtoolize: creating file './configure'
libtoolize: copying file './libltdl/lt__alloc.h'
libtoolize: copying file './libltdl/lt__argz_.h'
libtoolize: copying file './libltdl/lt__dirent.h'
libtoolize: copying file './libltdl/lt__glibc.h'
libtoolize: copying file './libltdl/lt__private.h'
libtoolize: copying file './libltdl/lt__strl.h'
libtoolize: copying file './libltdl/lt_dlloader.h'
libtoolize: copying file './libltdl/lt_error.h'
libtoolize: copying file './libltdl/lt_system.h'
libtoolize: copying file './libltdl/slist.h'
libtoolize: copying file './loaders/dld_link.c'
libtoolize: copying file './loaders/dlopen.c'
libtoolize: copying file './loaders/dyld.c'
libtoolize: copying file './loaders/load_add_on.c'
libtoolize: copying file './loaders/loadlibrary.c'
libtoolize: copying file './loaders/preopen.c'
libtoolize: copying file './loaders/shl_load.c'
libtoolize: copying file './lt__alloc.c'
libtoolize: copying file './lt__argz.c'
libtoolize: copying file './lt__dirent.c'
libtoolize: copying file './lt__strl.c'
libtoolize: copying file './lt_dlloader.c'
libtoolize: copying file './lt_error.c'
libtoolize: copying file './ltdl.c'
libtoolize: copying file './ltdl.h'
libtoolize: copying file './slist.c'
libtoolize: Remember to add 'LTDL_INIT' to configure.ac.
libtoolize: Consider adding 'AC_CONFIG_MACRO_DIRS([m4])' to configure.ac,
libtoolize: and rerunning libtoolize and aclocal.
libtoolize: putting auxiliary files in '.'.
libtoolize: copying file './compile'
libtoolize: copying file './config.guess'
libtoolize: copying file './config.sub'
libtoolize: copying file './depcomp'
libtoolize: copying file './install-sh'
libtoolize: copying file './missing'
libtoolize: copying file './ltmain.sh'
libtoolize: putting macros in 'libltdl/m4'.
libtoolize: copying file 'libltdl/m4/libtool.m4'
libtoolize: You should add the contents of 'libltdl/m4/libtool.m4' to 'aclocal.m4'.
libtoolize: copying file 'libltdl/m4/ltargz.m4'
libtoolize: You should add the contents of 'libltdl/m4/ltargz.m4' to 'aclocal.m4'.
libtoolize: copying file 'libltdl/m4/ltdl.m4'
libtoolize: You should add the contents of 'libltdl/m4/ltdl.m4' to 'aclocal.m4'.
libtoolize: copying file 'libltdl/m4/ltoptions.m4'
libtoolize: You should add the contents of 'libltdl/m4/ltoptions.m4' to 'aclocal.m4'.
libtoolize: copying file 'libltdl/m4/ltsugar.m4'
libtoolize: You should add the contents of 'libltdl/m4/ltsugar.m4' to 'aclocal.m4'.
libtoolize: copying file 'libltdl/m4/ltversion.m4'
libtoolize: copying file 'libltdl/m4/lt~obsolete.m4'
libtoolize: You should add the contents of 'libltdl/m4/lt~obsolete.m4' to 'aclocal.m4'.
libtoolize: putting libltdl files in LT_CONFIG_LTDL_DIR, 'libltdl'.
libtoolize: copying file 'libltdl/COPYING.LIB'
libtoolize: creating file 'libltdl/Makefile.am'
libtoolize: copying file 'libltdl/README'
libtoolize: creating file 'libltdl/configure.ac'
libtoolize: creating file 'libltdl/aclocal.m4'
libtoolize: creating file 'libltdl/Makefile.in'
libtoolize: copying file 'libltdl/config-h.in'
libtoolize: creating file 'libltdl/configure'
libtoolize: copying file 'libltdl/libltdl/lt__alloc.h'
libtoolize: copying file 'libltdl/libltdl/lt__argz_.h'
libtoolize: copying file 'libltdl/libltdl/lt__dirent.h'
libtoolize: copying file 'libltdl/libltdl/lt__glibc.h'
libtoolize: copying file 'libltdl/libltdl/lt__private.h'
libtoolize: copying file 'libltdl/libltdl/lt__strl.h'
libtoolize: copying file 'libltdl/libltdl/lt_dlloader.h'
libtoolize: copying file 'libltdl/libltdl/lt_error.h'
libtoolize: copying file 'libltdl/libltdl/lt_system.h'
libtoolize: copying file 'libltdl/libltdl/slist.h'
libtoolize: copying file 'libltdl/loaders/dld_link.c'
libtoolize: copying file 'libltdl/loaders/dlopen.c'
libtoolize: copying file 'libltdl/loaders/dyld.c'
libtoolize: copying file 'libltdl/loaders/load_add_on.c'
libtoolize: copying file 'libltdl/loaders/loadlibrary.c'
libtoolize: copying file 'libltdl/loaders/preopen.c'
libtoolize: copying file 'libltdl/loaders/shl_load.c'
libtoolize: copying file 'libltdl/lt__alloc.c'
libtoolize: copying file 'libltdl/lt__argz.c'
libtoolize: copying file 'libltdl/lt__dirent.c'
libtoolize: copying file 'libltdl/lt__strl.c'
libtoolize: copying file 'libltdl/lt_dlloader.c'
libtoolize: copying file 'libltdl/lt_error.c'
libtoolize: copying file 'libltdl/ltdl.c'
libtoolize: copying file 'libltdl/ltdl.h'
libtoolize: copying file 'libltdl/slist.c'
libtoolize: You should add the contents of the following files to 'aclocal.m4':
libtoolize:   '/usr/share/aclocal/ltargz.m4'
libtoolize: Remember to add 'LTDL_INIT' to configure.ac.
libtoolize: Consider adding 'AC_CONFIG_MACRO_DIRS([libltdl/m4])' to configure.ac,
libtoolize: and rerunning libtoolize and aclocal.
libtoolize: Consider adding '-I m4' to ACLOCAL_AMFLAGS in Makefile.am.

$ ./configure --prefix=/scratch/bsmith/petsc/arch-linux-c-debug MAKE=/usr/bin/make --libdir=/scratch/bsmith/petsc/arch-linux-c-debug/lib CC="/scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc" CFLAGS="-fstack-protector -g3 -O0 -fopenmp" AR="/usr/bin/ar" ARFLAGS="cr" LDFLAGS=" -fopenmp" CXX="/scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicxx" CXXFLAGS=" -g -O0 -fstack-protector -g -O0 -fopenmp" F90="/scratch/bsmith/petsc/arch-linux-c-debug/bin/mpif90" F90FLAGS="-ffree-line-length-0 -g -O0 -fopenmp" FFLAGS="-ffree-line-length-0 -g -O0 -fopenmp" FC="/scratch/bsmith/petsc/arch-linux-c-debug/bin/mpif90" F77="/scratch/bsmith/petsc/arch-linux-c-debug/bin/mpif90" FCFLAGS="-ffree-line-length-0 -g -O0 -fopenmp" --enable-shared --with-mpi=/scratch/bsmith/petsc/arch-linux-c-debug --with-python=3 --with-mpi=/scratch/bsmith/petsc/arch-linux-c-debug
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /bin/mkdir -p
checking for gawk... gawk
checking whether /usr/bin/make sets $(MAKE)... yes
checking whether /usr/bin/make supports nested variables... yes
checking whether to enable maintainer-specific portions of Makefiles... no
checking build system type... x86_64-pc-linux-gnu
checking host system type... x86_64-pc-linux-gnu
checking whether we are configuring for Blue Gene... no
checking for style of include used by /usr/bin/make... GNU
checking for gcc... /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc accepts -g... yes
checking for /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc option to accept ISO C89... none needed
checking whether /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc understands -c and -o together... yes
checking dependency style of /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc... gcc3
checking how to run the C preprocessor... /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc -E
checking for grep that handles long lines and -e... /bin/grep
checking for egrep... /bin/grep -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking minix/config.h usability... no
checking minix/config.h presence... no
checking for minix/config.h... no
checking whether it is safe to define __EXTENSIONS__... yes
checking for gcc... (cached) /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc
checking whether we are using the GNU C compiler... (cached) yes
checking whether /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc accepts -g... (cached) yes
checking for /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc option to accept ISO C89... (cached) none needed
checking whether /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc understands -c and -o together... (cached) yes
checking dependency style of /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc... (cached) gcc3
checking whether we are using the GNU C++ compiler... yes
checking whether /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicxx accepts -g... yes
checking dependency style of /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicxx... gcc3
checking for a Python interpreter with version >= "3"... python
checking for python... /scratch/bsmith/python/bin/python
checking for python version... 3.9
checking for python platform... linux
checking for python script directory... ${prefix}/lib/python3.9/site-packages
checking for python extension module directory... ${exec_prefix}/lib/python3.9/site-packages
Platform: x86_64-pc-linux-gnu
Compiler : gcc
Using AM_CFLAGS=   -W -Wall -pedantic -Wno-long-long -g -O3 -fstack-protector -g3 -O0 -fopenmp
Platform: x86_64-pc-linux-gnu
Compiler : g++
Using AM_CXXFLAGS=   -W -Wall -pedantic -Wno-long-long -g -O3  -g -O0 -fstack-protector -g -O0 -fopenmp
Using AM_LDFLAGS=  -fopenmp
checking for a sed that does not truncate output... /bin/sed
checking for fgrep... /bin/grep -F
checking how to print strings... printf
checking for ld used by /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc... /usr/bin/ld
checking if the linker (/usr/bin/ld) is GNU ld... yes
checking for BSD- or MS-compatible name lister (nm)... /usr/bin/nm -B
checking the name lister (/usr/bin/nm -B) interface... BSD nm
checking whether ln -s works... yes
checking the maximum length of command line arguments... 1572864
checking how to convert x86_64-pc-linux-gnu file names to x86_64-pc-linux-gnu format... func_convert_file_noop
checking how to convert x86_64-pc-linux-gnu file names to toolchain format... func_convert_file_noop
checking for /usr/bin/ld option to reload object files... -r
checking for objdump... objdump
checking how to recognize dependent libraries... pass_all
checking for dlltool... no
checking how to associate runtime and link libraries... printf %s\n
checking for archiver @FILE support... @
checking for strip... strip
checking for ranlib... ranlib
checking command to parse /usr/bin/nm -B output from /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc object... ok
checking for sysroot... no
checking for a working dd... /bin/dd
checking how to truncate binary pipes... /bin/dd bs=4096 count=1
checking for mt... mt
checking if mt is a manifest tool... no
checking for dlfcn.h... yes
checking for objdir... .libs
checking if /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc supports -fno-rtti -fno-exceptions... no
checking for /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc option to produce PIC... -fPIC -DPIC
checking if /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc PIC flag -fPIC -DPIC works... yes
checking if /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc static flag -static works... yes
checking if /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc supports -c -o file.o... yes
checking if /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc supports -c -o file.o... (cached) yes
checking whether the /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking whether -lc should be explicitly linked in... no
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking for shl_load... no
checking for shl_load in -ldld... no
checking for dlopen... no
checking for dlopen in -ldl... yes
checking whether a program can dlopen itself... yes
checking whether a statically linked program can dlopen itself... no
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... yes
checking how to run the C++ preprocessor... /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicxx -E
checking for ld used by /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicxx... /usr/bin/ld -m elf_x86_64
checking if the linker (/usr/bin/ld -m elf_x86_64) is GNU ld... yes
checking whether the /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicxx linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking for /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicxx option to produce PIC... -fPIC -DPIC
checking if /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicxx PIC flag -fPIC -DPIC works... yes
checking if /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicxx static flag -static works... yes
checking if /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicxx supports -c -o file.o... yes
checking if /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicxx supports -c -o file.o... (cached) yes
checking whether the /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicxx linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking dynamic linker characteristics... (cached) GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking whether /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicxx links MPI programs out of the box... yes
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: creating neurosim/Makefile
config.status: creating neurosim/config.h
config.status: creating neurosim/version.h
config.status: creating neurosim/examples/Makefile
config.status: creating libpyneurosim/Makefile
config.status: creating config.h
config.status: executing depfiles commands
config.status: executing libtool commands
=== configuring in libltdl (/scratch/bsmith/petsc/arch-linux-c-debug/externalpackages/git.libneurosim/libltdl)
configure: running /bin/bash ./configure --disable-option-checking '--prefix=/scratch/bsmith/petsc/arch-linux-c-debug'  'MAKE=/usr/bin/make' '--libdir=/scratch/bsmith/petsc/arch-linux-c-debug/lib' 'CC=/scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc' 'CFLAGS=-fstack-protector -g3 -O0 -fopenmp' 'AR=/usr/bin/ar' 'ARFLAGS=cr' 'LDFLAGS= -fopenmp' 'CXX=/scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicxx' 'CXXFLAGS= -g -O0 -fstack-protector -g -O0 -fopenmp' 'F90=/scratch/bsmith/petsc/arch-linux-c-debug/bin/mpif90' 'F90FLAGS=-ffree-line-length-0 -g -O0 -fopenmp' 'FFLAGS=-ffree-line-length-0 -g -O0 -fopenmp' 'FC=/scratch/bsmith/petsc/arch-linux-c-debug/bin/mpif90' 'F77=/scratch/bsmith/petsc/arch-linux-c-debug/bin/mpif90' 'FCFLAGS=-ffree-line-length-0 -g -O0 -fopenmp' '--enable-shared' '--with-python=3' '--with-mpi=/scratch/bsmith/petsc/arch-linux-c-debug' '--enable-ltdl-convenience' --cache-file=/dev/null --srcdir=.
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /bin/mkdir -p
checking for gawk... gawk
checking whether /usr/bin/make sets $(MAKE)... yes
checking whether /usr/bin/make supports nested variables... yes
checking whether /usr/bin/make supports nested variables... (cached) yes
checking build system type... x86_64-pc-linux-gnu
checking host system type... x86_64-pc-linux-gnu
checking how to print strings... printf
checking for style of include used by /usr/bin/make... GNU
checking for gcc... /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc accepts -g... yes
checking for /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc option to accept ISO C89... none needed
checking whether /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc understands -c and -o together... yes
checking dependency style of /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc... gcc3
checking for a sed that does not truncate output... /bin/sed
checking for grep that handles long lines and -e... /bin/grep
checking for egrep... /bin/grep -E
checking for fgrep... /bin/grep -F
checking for ld used by /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc... /usr/bin/ld
checking if the linker (/usr/bin/ld) is GNU ld... yes
checking for BSD- or MS-compatible name lister (nm)... /usr/bin/nm -B
checking the name lister (/usr/bin/nm -B) interface... BSD nm
checking whether ln -s works... yes
checking the maximum length of command line arguments... 1572864
checking how to convert x86_64-pc-linux-gnu file names to x86_64-pc-linux-gnu format... func_convert_file_noop
checking how to convert x86_64-pc-linux-gnu file names to toolchain format... func_convert_file_noop
checking for /usr/bin/ld option to reload object files... -r
checking for objdump... objdump
checking how to recognize dependent libraries... pass_all
checking for dlltool... no
checking how to associate runtime and link libraries... printf %s\n
checking for archiver @FILE support... @
checking for strip... strip
checking for ranlib... ranlib
checking command to parse /usr/bin/nm -B output from /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc object... ok
checking for sysroot... no
checking for a working dd... /bin/dd
checking how to truncate binary pipes... /bin/dd bs=4096 count=1
checking for mt... mt
checking if mt is a manifest tool... no
checking how to run the C preprocessor... /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking for dlfcn.h... yes
checking for objdir... .libs
checking if /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc supports -fno-rtti -fno-exceptions... no
checking for /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc option to produce PIC... -fPIC -DPIC
checking if /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc PIC flag -fPIC -DPIC works... yes
checking if /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc static flag -static works... yes
checking if /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc supports -c -o file.o... yes
checking if /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc supports -c -o file.o... (cached) yes
checking whether the /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking whether -lc should be explicitly linked in... no
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking for shl_load... no
checking for shl_load in -ldld... no
checking for dlopen... no
checking for dlopen in -ldl... yes
checking whether a program can dlopen itself... yes
checking whether a statically linked program can dlopen itself... no
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... yes
checking what extension is used for runtime loadable modules... .so
checking what variable specifies run-time module search path... LD_LIBRARY_PATH
checking for the default library search path... /lib /usr/lib /usr/lib/x86_64-linux-gnu/libfakeroot /usr/local/lib /usr/local/lib/x86_64-linux-gnu /lib/x86_64-linux-gnu /usr/lib/x86_64-linux-gnu /lib32 /usr/lib32 
checking for library containing dlopen... -ldl
checking for dlerror... yes
checking for shl_load... (cached) no
checking for shl_load in -ldld... (cached) no
checking for dld_link in -ldld... no
checking for _ prefix in compiled symbols... no
checking whether deplibs are loaded by dlopen... yes
checking for argz.h... yes
checking for error_t... yes
checking for argz_add... yes
checking for argz_append... yes
checking for argz_count... yes
checking for argz_create_sep... yes
checking for argz_insert... yes
checking for argz_next... yes
checking for argz_stringify... yes
checking if argz actually works... yes
checking whether libtool supports -dlopen/-dlpreopen... yes
checking for unistd.h... (cached) yes
checking for dl.h... no
checking for sys/dl.h... no
checking for dld.h... no
checking for mach-o/dyld.h... no
checking for dirent.h... yes
checking for closedir... yes
checking for opendir... yes
checking for readdir... yes
checking for strlcat... no
checking for strlcpy... no
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: creating config.h
config.status: executing depfiles commands
config.status: executing libtool commands

$ make 
make  all-recursive
make[1]: Entering directory '/scratch/bsmith/petsc/arch-linux-c-debug/externalpackages/git.libneurosim'
Making all in libltdl
make[2]: Entering directory '/scratch/bsmith/petsc/arch-linux-c-debug/externalpackages/git.libneurosim/libltdl'
CDPATH="${ZSH_VERSION+.}:" && cd . && /bin/bash /scratch/bsmith/petsc/arch-linux-c-debug/externalpackages/git.libneurosim/missing aclocal-1.15 -I m4
 cd . && /bin/bash /scratch/bsmith/petsc/arch-linux-c-debug/externalpackages/git.libneurosim/missing automake-1.15 --foreign
CDPATH="${ZSH_VERSION+.}:" && cd . && /bin/bash /scratch/bsmith/petsc/arch-linux-c-debug/externalpackages/git.libneurosim/missing autoconf
/bin/bash ./config.status --recheck
running CONFIG_SHELL=/bin/bash /bin/bash ./configure --disable-option-checking --prefix=/scratch/bsmith/petsc/arch-linux-c-debug MAKE=/usr/bin/make --libdir=/scratch/bsmith/petsc/arch-linux-c-debug/lib CC=/scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc CFLAGS=-fstack-protector -g3 -O0 -fopenmp AR=/usr/bin/ar ARFLAGS=cr LDFLAGS= -fopenmp CXX=/scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicxx CXXFLAGS= -g -O0 -fstack-protector -g -O0 -fopenmp F90=/scratch/bsmith/petsc/arch-linux-c-debug/bin/mpif90 F90FLAGS=-ffree-line-length-0 -g -O0 -fopenmp FFLAGS=-ffree-line-length-0 -g -O0 -fopenmp FC=/scratch/bsmith/petsc/arch-linux-c-debug/bin/mpif90 F77=/scratch/bsmith/petsc/arch-linux-c-debug/bin/mpif90 FCFLAGS=-ffree-line-length-0 -g -O0 -fopenmp --enable-shared --with-python=3 --with-mpi=/scratch/bsmith/petsc/arch-linux-c-debug --enable-ltdl-convenience --cache-file=/dev/null --srcdir=. --no-create --no-recursion
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /bin/mkdir -p
checking for gawk... gawk
checking whether /usr/bin/make sets $(MAKE)... yes
checking whether /usr/bin/make supports nested variables... yes
checking whether /usr/bin/make supports nested variables... (cached) yes
checking build system type... x86_64-pc-linux-gnu
checking host system type... x86_64-pc-linux-gnu
checking how to print strings... printf
checking for style of include used by /usr/bin/make... GNU
checking for gcc... /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc accepts -g... yes
checking for /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc option to accept ISO C89... none needed
checking whether /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc understands -c and -o together... yes
checking dependency style of /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc... gcc3
checking for a sed that does not truncate output... /bin/sed
checking for grep that handles long lines and -e... /bin/grep
checking for egrep... /bin/grep -E
checking for fgrep... /bin/grep -F
checking for ld used by /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc... /usr/bin/ld
checking if the linker (/usr/bin/ld) is GNU ld... yes
checking for BSD- or MS-compatible name lister (nm)... /usr/bin/nm -B
checking the name lister (/usr/bin/nm -B) interface... BSD nm
checking whether ln -s works... yes
checking the maximum length of command line arguments... 1572864
checking how to convert x86_64-pc-linux-gnu file names to x86_64-pc-linux-gnu format... func_convert_file_noop
checking how to convert x86_64-pc-linux-gnu file names to toolchain format... func_convert_file_noop
checking for /usr/bin/ld option to reload object files... -r
checking for objdump... objdump
checking how to recognize dependent libraries... pass_all
checking for dlltool... no
checking how to associate runtime and link libraries... printf %s\n
checking for archiver @FILE support... @
checking for strip... strip
checking for ranlib... ranlib
checking command to parse /usr/bin/nm -B output from /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc object... ok
checking for sysroot... no
checking for a working dd... /bin/dd
checking how to truncate binary pipes... /bin/dd bs=4096 count=1
checking for mt... mt
checking if mt is a manifest tool... no
checking how to run the C preprocessor... /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking for dlfcn.h... yes
checking for objdir... .libs
checking if /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc supports -fno-rtti -fno-exceptions... no
checking for /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc option to produce PIC... -fPIC -DPIC
checking if /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc PIC flag -fPIC -DPIC works... yes
checking if /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc static flag -static works... yes
checking if /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc supports -c -o file.o... yes
checking if /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc supports -c -o file.o... (cached) yes
checking whether the /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking whether -lc should be explicitly linked in... no
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking for shl_load... no
checking for shl_load in -ldld... no
checking for dlopen... no
checking for dlopen in -ldl... yes
checking whether a program can dlopen itself... yes
checking whether a statically linked program can dlopen itself... no
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... yes
checking what extension is used for runtime loadable modules... .so
checking what variable specifies run-time module search path... LD_LIBRARY_PATH
checking for the default library search path... /lib /usr/lib /usr/lib/x86_64-linux-gnu/libfakeroot /usr/local/lib /usr/local/lib/x86_64-linux-gnu /lib/x86_64-linux-gnu /usr/lib/x86_64-linux-gnu /lib32 /usr/lib32 
checking for library containing dlopen... -ldl
checking for dlerror... yes
checking for shl_load... (cached) no
checking for shl_load in -ldld... (cached) no
checking for dld_link in -ldld... no
checking for _ prefix in compiled symbols... no
checking whether deplibs are loaded by dlopen... yes
checking for argz.h... yes
checking for error_t... yes
checking for argz_add... yes
checking for argz_append... yes
checking for argz_count... yes
checking for argz_create_sep... yes
checking for argz_insert... yes
checking for argz_next... yes
checking for argz_stringify... yes
checking if argz actually works... yes
checking whether libtool supports -dlopen/-dlpreopen... yes
checking for unistd.h... (cached) yes
checking for dl.h... no
checking for sys/dl.h... no
checking for dld.h... no
checking for mach-o/dyld.h... no
checking for dirent.h... yes
checking for closedir... yes
checking for opendir... yes
checking for readdir... yes
checking for strlcat... no
checking for strlcpy... no
checking that generated files are newer than configure... done
configure: creating ./config.status
 /bin/bash ./config.status
config.status: creating Makefile
config.status: creating config.h
config.status: config.h is unchanged
config.status: executing depfiles commands
config.status: executing libtool commands
(CDPATH="${ZSH_VERSION+.}:" && cd . && /bin/bash /scratch/bsmith/petsc/arch-linux-c-debug/externalpackages/git.libneurosim/missing autoheader)
rm -f stamp-h1
touch config-h.in
cd . && /bin/bash ./config.status config.h
config.status: creating config.h
config.status: config.h is unchanged
make  all-am
make[3]: Entering directory '/scratch/bsmith/petsc/arch-linux-c-debug/externalpackages/git.libneurosim/libltdl'
  CC       loaders/dlopen.lo
  CCLD     dlopen.la
/usr/bin/ar: `u' modifier ignored since `D' is the default (see `U')
  CC       loaders/libltdlc_la-preopen.lo
  CC       libltdlc_la-lt__alloc.lo
  CC       libltdlc_la-lt_dlloader.lo
  CC       libltdlc_la-lt_error.lo
  CC       libltdlc_la-ltdl.lo
  CC       libltdlc_la-slist.lo
  CC       lt__strl.lo
  CCLD     libltdlc.la
/usr/bin/ar: `u' modifier ignored since `D' is the default (see `U')
make[3]: Leaving directory '/scratch/bsmith/petsc/arch-linux-c-debug/externalpackages/git.libneurosim/libltdl'
make[2]: Leaving directory '/scratch/bsmith/petsc/arch-linux-c-debug/externalpackages/git.libneurosim/libltdl'
Making all in neurosim
make[2]: Entering directory '/scratch/bsmith/petsc/arch-linux-c-debug/externalpackages/git.libneurosim/neurosim'
Making all in .
make[3]: Entering directory '/scratch/bsmith/petsc/arch-linux-c-debug/externalpackages/git.libneurosim/neurosim'
depbase=`echo connection_generator.lo | sed 's|[^/]*$|.deps/&|;s|\.lo$||'`;\
/bin/bash ../libtool  --tag=CXX   --mode=compile /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicxx -DHAVE_CONFIG_H -I. -I..  -I.. -I../libltdl   -W -Wall -pedantic -Wno-long-long -g -O3  -g -O0 -fstack-protector -g -O0 -fopenmp  -MT connection_generator.lo -MD -MP -MF $depbase.Tpo -c -o connection_generator.lo connection_generator.cpp &&\
mv -f $depbase.Tpo $depbase.Plo
libtool: compile:  /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicxx -DHAVE_CONFIG_H -I. -I.. -I.. -I../libltdl -W -Wall -pedantic -Wno-long-long -g -O3 -g -O0 -fstack-protector -g -O0 -fopenmp -MT connection_generator.lo -MD -MP -MF .deps/connection_generator.Tpo -c connection_generator.cpp  -fPIC -DPIC -o .libs/connection_generator.o
libtool: compile:  /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicxx -DHAVE_CONFIG_H -I. -I.. -I.. -I../libltdl -W -Wall -pedantic -Wno-long-long -g -O3 -g -O0 -fstack-protector -g -O0 -fopenmp -MT connection_generator.lo -MD -MP -MF .deps/connection_generator.Tpo -c connection_generator.cpp -o connection_generator.o >/dev/null 2>&1
depbase=`echo connection_generator_V2_0.lo | sed 's|[^/]*$|.deps/&|;s|\.lo$||'`;\
/bin/bash ../libtool  --tag=CXX   --mode=compile /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicxx -DHAVE_CONFIG_H -I. -I..  -I.. -I../libltdl   -W -Wall -pedantic -Wno-long-long -g -O3  -g -O0 -fstack-protector -g -O0 -fopenmp  -MT connection_generator_V2_0.lo -MD -MP -MF $depbase.Tpo -c -o connection_generator_V2_0.lo connection_generator_V2_0.cpp &&\
mv -f $depbase.Tpo $depbase.Plo
libtool: compile:  /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicxx -DHAVE_CONFIG_H -I. -I.. -I.. -I../libltdl -W -Wall -pedantic -Wno-long-long -g -O3 -g -O0 -fstack-protector -g -O0 -fopenmp -MT connection_generator_V2_0.lo -MD -MP -MF .deps/connection_generator_V2_0.Tpo -c connection_generator_V2_0.cpp  -fPIC -DPIC -o .libs/connection_generator_V2_0.o
libtool: compile:  /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicxx -DHAVE_CONFIG_H -I. -I.. -I.. -I../libltdl -W -Wall -pedantic -Wno-long-long -g -O3 -g -O0 -fstack-protector -g -O0 -fopenmp -MT connection_generator_V2_0.lo -MD -MP -MF .deps/connection_generator_V2_0.Tpo -c connection_generator_V2_0.cpp -o connection_generator_V2_0.o >/dev/null 2>&1
depbase=`echo version.lo | sed 's|[^/]*$|.deps/&|;s|\.lo$||'`;\
/bin/bash ../libtool  --tag=CC   --mode=compile /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc -DHAVE_CONFIG_H -I. -I..  -I.. -I../libltdl   -W -Wall -pedantic -Wno-long-long -g -O3 -fstack-protector -g3 -O0 -fopenmp  -MT version.lo -MD -MP -MF $depbase.Tpo -c -o version.lo version.c &&\
mv -f $depbase.Tpo $depbase.Plo
libtool: compile:  /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc -DHAVE_CONFIG_H -I. -I.. -I.. -I../libltdl -W -Wall -pedantic -Wno-long-long -g -O3 -fstack-protector -g3 -O0 -fopenmp -MT version.lo -MD -MP -MF .deps/version.Tpo -c version.c  -fPIC -DPIC -o .libs/version.o
libtool: compile:  /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicc -DHAVE_CONFIG_H -I. -I.. -I.. -I../libltdl -W -Wall -pedantic -Wno-long-long -g -O3 -fstack-protector -g3 -O0 -fopenmp -MT version.lo -MD -MP -MF .deps/version.Tpo -c version.c -o version.o >/dev/null 2>&1
/bin/bash ../libtool  --tag=CXX   --mode=link /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicxx -W -Wall -pedantic -Wno-long-long -g -O3  -g -O0 -fstack-protector -g -O0 -fopenmp    -o libneurosim.la -rpath /scratch/bsmith/petsc/arch-linux-c-debug/lib connection_generator.lo connection_generator_V2_0.lo version.lo ../libltdl/libltdlc.la   
libtool: link: /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicxx  -fPIC -DPIC -shared -nostdlib /usr/lib/x86_64-linux-gnu/crti.o /nfs/gce/software/spack/opt/spack/linux-ubuntu18.04-x86_64/gcc-7.3.0/gcc-8.3.0-fjpc5ys/lib/gcc/x86_64-pc-linux-gnu/8.3.0/crtbeginS.o  .libs/connection_generator.o .libs/connection_generator_V2_0.o .libs/version.o  -Wl,--whole-archive ../libltdl/.libs/libltdlc.a -Wl,--no-whole-archive  -Wl,-rpath -Wl,/scratch/bsmith/petsc/arch-linux-c-debug/lib -Wl,-rpath -Wl,/nfs/gce/software/spack/opt/spack/linux-ubuntu18.04-x86_64/gcc-7.3.0/gcc-8.3.0-fjpc5ys/lib/../lib64 -Wl,-rpath -Wl,/scratch/bsmith/petsc/arch-linux-c-debug/lib -Wl,-rpath -Wl,/nfs/gce/software/spack/opt/spack/linux-ubuntu18.04-x86_64/gcc-7.3.0/gcc-8.3.0-fjpc5ys/lib/../lib64 -ldl -L/scratch/bsmith/petsc/arch-linux-c-debug/lib -L/nfs/gce/software/spack/opt/spack/linux-ubuntu18.04-x86_64/gcc-7.3.0/gcc-8.3.0-fjpc5ys/lib/gcc/x86_64-pc-linux-gnu/8.3.0 -L/nfs/gce/software/spack/opt/spack/linux-ubuntu18.04-x86_64/gcc-7.3.0/gcc-8.3.0-fjpc5ys/lib/gcc/x86_64-pc-linux-gnu/8.3.0/../../../../lib64 -L/lib/x86_64-linux-gnu -L/lib/../lib64 -L/usr/lib/x86_64-linux-gnu -L/nfs/gce/software/spack/opt/spack/linux-ubuntu18.04-x86_64/gcc-7.3.0/gcc-8.3.0-fjpc5ys/lib/gcc/x86_64-pc-linux-gnu/8.3.0/../../.. /scratch/bsmith/petsc/arch-linux-c-debug/lib/libmpicxx.so /scratch/bsmith/petsc/arch-linux-c-debug/lib/libmpi.so /nfs/gce/software/spack/opt/spack/linux-ubuntu18.04-x86_64/gcc-7.3.0/gcc-8.3.0-fjpc5ys/lib/../lib64/libstdc++.so -lm -lc -lgcc_s /nfs/gce/software/spack/opt/spack/linux-ubuntu18.04-x86_64/gcc-7.3.0/gcc-8.3.0-fjpc5ys/lib/gcc/x86_64-pc-linux-gnu/8.3.0/crtendS.o /usr/lib/x86_64-linux-gnu/crtn.o  -g -O3 -g -O0 -fstack-protector -g -O0 -fopenmp   -fopenmp -Wl,-soname -Wl,libneurosim.so.0 -o .libs/libneurosim.so.0.0.0
libtool: link: (cd ".libs" && rm -f "libneurosim.so.0" && ln -s "libneurosim.so.0.0.0" "libneurosim.so.0")
libtool: link: (cd ".libs" && rm -f "libneurosim.so" && ln -s "libneurosim.so.0.0.0" "libneurosim.so")
libtool: link: (cd .libs/libneurosim.lax/libltdlc.a && /usr/bin/ar x "/scratch/bsmith/petsc/arch-linux-c-debug/externalpackages/git.libneurosim/neurosim/../libltdl/.libs/libltdlc.a")
libtool: link: /usr/bin/ar cru .libs/libneurosim.a  connection_generator.o connection_generator_V2_0.o version.o  .libs/libneurosim.lax/libltdlc.a/dlopen.o .libs/libneurosim.lax/libltdlc.a/libltdlcS.o .libs/libneurosim.lax/libltdlc.a/libltdlc_la-lt__alloc.o .libs/libneurosim.lax/libltdlc.a/libltdlc_la-lt_dlloader.o .libs/libneurosim.lax/libltdlc.a/libltdlc_la-lt_error.o .libs/libneurosim.lax/libltdlc.a/libltdlc_la-ltdl.o .libs/libneurosim.lax/libltdlc.a/libltdlc_la-preopen.o .libs/libneurosim.lax/libltdlc.a/libltdlc_la-slist.o .libs/libneurosim.lax/libltdlc.a/lt__strl.o 
/usr/bin/ar: `u' modifier ignored since `D' is the default (see `U')
libtool: link: ranlib .libs/libneurosim.a
libtool: link: rm -fr .libs/libneurosim.lax
libtool: link: ( cd ".libs" && rm -f "libneurosim.la" && ln -s "../libneurosim.la" "libneurosim.la" )
make[3]: Leaving directory '/scratch/bsmith/petsc/arch-linux-c-debug/externalpackages/git.libneurosim/neurosim'
Making all in examples
make[3]: Entering directory '/scratch/bsmith/petsc/arch-linux-c-debug/externalpackages/git.libneurosim/neurosim/examples'
/scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicxx -DHAVE_CONFIG_H -I. -I../..  -I../..  -W -Wall -pedantic -Wno-long-long -g -O3  -g -O0 -fstack-protector -g -O0 -fopenmp  -MT runiter1.o -MD -MP -MF .deps/runiter1.Tpo -c -o runiter1.o runiter1.cpp
mv -f .deps/runiter1.Tpo .deps/runiter1.Po
/scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicxx -DHAVE_CONFIG_H -I. -I../..  -I../..  -W -Wall -pedantic -Wno-long-long -g -O3  -g -O0 -fstack-protector -g -O0 -fopenmp  -MT simulator-model.o -MD -MP -MF .deps/simulator-model.Tpo -c -o simulator-model.o simulator-model.cpp
mv -f .deps/simulator-model.Tpo .deps/simulator-model.Po
/bin/bash ../../libtool  --tag=CXX   --mode=link /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicxx -W -Wall -pedantic -Wno-long-long -g -O3  -g -O0 -fstack-protector -g -O0 -fopenmp    -o runiter1 runiter1.o simulator-model.o ../../neurosim/libneurosim.la 
libtool: link: /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicxx -W -Wall -pedantic -Wno-long-long -g -O3 -g -O0 -fstack-protector -g -O0 -fopenmp -o .libs/runiter1 runiter1.o simulator-model.o  ../../neurosim/.libs/libneurosim.so -fopenmp -Wl,-rpath -Wl,/scratch/bsmith/petsc/arch-linux-c-debug/lib
/scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicxx -DHAVE_CONFIG_H -I. -I../..  -I../..  -W -Wall -pedantic -Wno-long-long -g -O3  -g -O0 -fstack-protector -g -O0 -fopenmp  -MT runiter2.o -MD -MP -MF .deps/runiter2.Tpo -c -o runiter2.o runiter2.cpp
mv -f .deps/runiter2.Tpo .deps/runiter2.Po
/bin/bash ../../libtool  --tag=CXX   --mode=link /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicxx -W -Wall -pedantic -Wno-long-long -g -O3  -g -O0 -fstack-protector -g -O0 -fopenmp    -o runiter2 runiter2.o simulator-model.o ../../neurosim/libneurosim.la 
libtool: link: /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicxx -W -Wall -pedantic -Wno-long-long -g -O3 -g -O0 -fstack-protector -g -O0 -fopenmp -o .libs/runiter2 runiter2.o simulator-model.o  ../../neurosim/.libs/libneurosim.so -fopenmp -Wl,-rpath -Wl,/scratch/bsmith/petsc/arch-linux-c-debug/lib
make[3]: Leaving directory '/scratch/bsmith/petsc/arch-linux-c-debug/externalpackages/git.libneurosim/neurosim/examples'
make[2]: Leaving directory '/scratch/bsmith/petsc/arch-linux-c-debug/externalpackages/git.libneurosim/neurosim'
Making all in libpyneurosim
make[2]: Entering directory '/scratch/bsmith/petsc/arch-linux-c-debug/externalpackages/git.libneurosim/libpyneurosim'
/bin/bash ../libtool  --tag=CXX   --mode=compile /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicxx -DHAVE_CONFIG_H -I. -I..    -I/scratch/bsmith/python/include/python3.9 -I./../libneurosim/conngen  -MT libpy3neurosim_la-pyneurosim.lo -MD -MP -MF .deps/libpy3neurosim_la-pyneurosim.Tpo -c -o libpy3neurosim_la-pyneurosim.lo `test -f 'pyneurosim.cpp' || echo './'`pyneurosim.cpp
libtool: compile:  /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicxx -DHAVE_CONFIG_H -I. -I.. -I/scratch/bsmith/python/include/python3.9 -I./../libneurosim/conngen -MT libpy3neurosim_la-pyneurosim.lo -MD -MP -MF .deps/libpy3neurosim_la-pyneurosim.Tpo -c pyneurosim.cpp  -fPIC -DPIC -o .libs/libpy3neurosim_la-pyneurosim.o
libtool: compile:  /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicxx -DHAVE_CONFIG_H -I. -I.. -I/scratch/bsmith/python/include/python3.9 -I./../libneurosim/conngen -MT libpy3neurosim_la-pyneurosim.lo -MD -MP -MF .deps/libpy3neurosim_la-pyneurosim.Tpo -c pyneurosim.cpp -o libpy3neurosim_la-pyneurosim.o >/dev/null 2>&1
mv -f .deps/libpy3neurosim_la-pyneurosim.Tpo .deps/libpy3neurosim_la-pyneurosim.Plo
/bin/bash ../libtool  --tag=CXX   --mode=link /scratch/bsmith/petsc/arch-linux-c-debug/bin/mpicxx -I/scratch/bsmith/python/include/python3.9 -I./../libneurosim/conngen  ../libneurosim/libneurosim.la  -o libpy3neurosim.la -rpath /scratch/bsmith/petsc/arch-linux-c-debug/lib libpy3neurosim_la-pyneurosim.lo  
libtool:   error: cannot find the library '../libneurosim/libneurosim.la' or unhandled argument '../libneurosim/libneurosim.la'
Makefile:518: recipe for target 'libpy3neurosim.la' failed
make[2]: *** [libpy3neurosim.la] Error 1
make[2]: Leaving directory '/scratch/bsmith/petsc/arch-linux-c-debug/externalpackages/git.libneurosim/libpyneurosim'
Makefile:424: recipe for target 'all-recursive' failed
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory '/scratch/bsmith/petsc/arch-linux-c-debug/externalpackages/git.libneurosim'
Makefile:356: recipe for target 'all' failed
make: *** [all] Error 2
```


```
$ uname -a
Linux petsc-01 4.15.0-121-generic #123-Ubuntu SMP Mon Oct 5 16:16:40 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
```

--- a/libpyneurosim/Makefile.am
+++ b/libpyneurosim/Makefile.am
@@ -13,8 +13,8 @@ libpy3neurosim_la_SOURCES = \
 libpy3neurosim_la_HEADERS = \
   pyneurosim.h
 
-libpy3neurosim_la_CXXFLAGS = -I@PYTHON_INCLUDE@ -I$(srcdir)/../libneurosim/conngen
-libpy3neurosim_la_LDFLAGS = $(top_builddir)/libneurosim/libneurosim.la
+libpy3neurosim_la_CXXFLAGS = -I@PYTHON_INCLUDE@ -I$(srcdir)/../neurosim/conngen
+libpy3neurosim_la_LDFLAGS = $(top_builddir)/neurosim/libneurosim.la
 libpy3neurosim_ladir = $(includedir)/neurosim
 else
 lib_LTLIBRARIES += libpy2neurosim.la libpyneurosim.la
@@ -25,8 +25,8 @@ libpy2neurosim_la_SOURCES = \
 libpy2neurosim_la_HEADERS = \
   pyneurosim.h
 
-libpy2neurosim_la_CXXFLAGS = -I@PYTHON_INCLUDE@ -I$(srcdir)/../libneurosim/conngen
-libpy2neurosim_la_LDFLAGS = $(top_builddir)/libneurosim/libneurosim.la
+libpy2neurosim_la_CXXFLAGS = -I@PYTHON_INCLUDE@ -I$(srcdir)/../neurosim/conngen
+libpy2neurosim_la_LDFLAGS = $(top_builddir)/neurosim/libneurosim.la
 libpy2neurosim_ladir = $(includedir)/neurosim
 # generate versionless version for backwards compatibility
 libpyneurosim_la_SOURCES = \

